cont3xt/db.js

```diff
@@ -355,6 +357,15 @@ class DbESImplementation {
     await this.client.indices.putMapping({
       index: 'cont3xt_overviews',
       body: {
+        properties: {
+          creator: { type: 'keyword' },
```

Comment:
I figured I would put mappings for the fields we know of explicitly—maybe that's better? If we're planning on adding 'type', should we be adding `type: { type: 'keyword' }` now, or once that's implemented, perhaps?

Reply:
sure